packages/kit/src/core/dev/plugin.js

```diff
@@ -163,8 +163,16 @@ export async function create_plugin(config) {
 
 			update_manifest();
 
-			vite.watcher.on('add', update_manifest);
-			vite.watcher.on('unlink', update_manifest);
+			console.error(`config.kit.files.routes: ${config.kit.files.routes}`);
```

Comment:
Are these console.errors leftovers of debugging?

Reply:
https://github.com/sveltejs/kit/pull/5076#discussion_r882054663